docs/versioned/serving/app-security/secure-pod-defaults.md

```diff
@@ -0,0 +1,89 @@
+---
+audience: administrator
+components:
+  - serving
+function: how-to
+---
+
+
+## SecurePodDefaults
+
+Knative Serving provides a `secure-pod-defaults` configuration option that gives operators granular control over pod security defaults, enabling progressive adoption of more secure pod configurations while maintaining backward compatibility with existing workloads. This feature offers three security levels—`disabled`, `root-allowed`, and `enabled` allowing organizations to gradually adopt security best practices without breaking container images that require specific permissions. The default is `disabled` to ensure existing deployments continue to work without modification.
+
+
+### Security Levels
+
+| Level | Description | Use Case |
+|-------|-------------|----------|
+| **disabled** | No security defaults applied | Legacy workloads, maximum compatibility |
+| **root-allowed** | Applies security defaults (blocks privilege escalation, runtime-default seccomp, drops capabilities) but allows root containers if not already set | Transition period, balanced security |
```

Comment:
nit: is there a way to disable word wrapping? 

<img width="169" height="120" alt="Screenshot 2025-10-21 at 4 28 00 PM" src="https://github.com/user-attachments/assets/0521757e-108d-4835-bee6-02526cd53076" />